content/docs/ref/comment.md

```diff
@@ -63,6 +63,27 @@ Any [generic option](/doc/ref) in addition to:
 
 ## Examples
 
+### Post a text report
+
+Write to your report using whatever method you prefer. For example, copy the
+contents of a text file containing the results of ML model training:
+
+```cli
+$ cat results.txt >> report.md
+```
+
+### Post a graphic report
+
+Display images using Markdown or HTML. Note that if an image is an output of
```

Comment:
Moved here from https://cml.dev/doc/usage#cml-reports

Reply:
please revert (or move to a separate PR - it has nothing to do with "Usage -> User Guide (initial)")